MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Use UpDown Button in .NET MAUI NumericEntry | Syncfusion
+description: Learn here all about how to use UpDown Button (SpinButton) in Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control and more.
+platform: MAUI
+control:  SfNumericEntry
+documentation: ug
+---
+
+# UpDown Button in .NET MAUI NumericEntry
+
+This section describes how to change value in the `NumericEntry` control using keys, mouse scrolling and the up-down button.
+
+## Increase or decrease value
+
+You can increment or decrement the value in the `NumericEntry` control by using **UpArrow**, **DownArrow**, **PageUp** and **PageDown** keys. You can change the increment or decrement value when Arrow keys are pressed, using the SmallChange property and for Page keys using the LargeChange property. By default, the value of `SmallChange` property is **1** and `LargeChange` property  is **10**.
```

Comment:
You can increment or decrement the value in the `NumericEntry` control using the **UpArrow**, **DownArrow**, **PageUp**, and **PageDown** keys. You can change the increment or decrement value when the Arrow keys are pressed using the SmallChange property and Page keys using the LargeChange property. By default, the value of the `SmallChange` property is **1**, and the `LargeChange` property is **10**.

Reply:
The requested content modification has been successfully addressed